src/test/java/de/mediathekview/mserver/crawler/wdr/tasks/WdrFilmDetailTaskTest.java

```diff
@@ -0,0 +1,256 @@
+package de.mediathekview.mserver.crawler.wdr.tasks;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.Assert.assertThat;
+
+import de.mediathekview.mlib.daten.Film;
+import de.mediathekview.mlib.daten.GeoLocations;
+import de.mediathekview.mlib.daten.Sender;
+import de.mediathekview.mserver.testhelper.AssertFilm;
+import de.mediathekview.mserver.testhelper.JsoupMock;
+import java.io.IOException;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import org.jsoup.Jsoup;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.modules.junit4.PowerMockRunnerDelegate;
+
+class TestData {
+  public String requestUrl;
+  public String filmPageFile;
+  public String jsUrl;
+  public String jsFile;
+  public String m3u8Url;
+  public String m3u8File;
+  public String topic;
+  public String expectedTitle;
+  public LocalDateTime expectedDate;
+  public Duration expectedDuration;
+  public String expectedDescription;
+  public String expectedSubtitle;
+  public String expectedUrlSmall;
+  public String expectedUrlNormal;
+  public String expectedUrlHd;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make expectedUrlHd a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)